rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/util/RMListenerProxy.java

```diff
@@ -124,6 +112,23 @@ public boolean init(String url, Credentials credentials) throws RMException, Key
         return true;
     }
 
+    /**
+     * @see org.ow2.proactive.resourcemanager.frontend.ResourceManager#disconnect()
+     */
+    public BooleanWrapper disconnect() {
+        try {
+            target.getMonitoring().removeRMEventListener();
+        } catch (RMException ignored) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)